packages/server/dataloader/customLoaderMakers.ts

```diff
@@ -406,6 +406,7 @@ export const meetingTemplatesByType = (parent: RootDataLoader) => {
       const resultsByType = await Promise.all(
         entries.map((entry) => {
           const [meetingType, teamIds] = entry
+          // Will convert to PG by Mar 1, 2023
```

Comment:
flagging all table reads with the same comment to alert other devs & make it easier to find instances later

Reply:
I'm treating older migrations as immutable & we'll just fix things up in a new migration, unless I'm missing something?

Reply:
I think it's fine. We only need to be careful how we delete the table. That's what I tried to describe. We **cannot** do
**rethink migrations**
* create `MeetingTemplate`
* multiple fill `MeetingTemplate`
* delete `MeetingTemplate`

**postgres migrations**
* create `MeetingTemplate`
* copy rethink `MeetingTemplate` data over

but instead we **need** to do
**rethink migrations**
* create `MeetingTemplate`
* multiple fill `MeetingTemplate`

**postgres migrations**
* create `MeetingTemplate`
* copy rethink `MeetingTemplate` data over
* delete rethink `MeetingTemplate`

Reply:
ohhh! yep, i get what you're saying. 💯  the `database/migrations` can no longer be touched, all migrations should live in `postgres/migrations`, even if they only touch rethinkdb in order to assure linearity. perhaps I should get rid of the rethinkdb `create migration` function to ensure it 😜